### PR TITLE
Rewrote if statements

### PR DESCRIPTION
**Pull request by debugduck**
From issues #7 and #8

### CODE REVIEW

File: PatternFinding.py [hunk 1]
```diff
@@ -129,11 +126,9 @@ def CheckingRatioOfContours(self, index):
         # print areaofsecondchild
         # print (areaoffirst/areaofsecondchild)
 
-        if (areaoffirst / areaofsecondchild) > 1 and \
-           ((areaoffirst / areaofsecondchild) < 10):
-            return True
-        else:
-            return False
+        return not ((areaoffirst / areaofsecondchild) > 1 and \
```
Comment (Griffintaur) [inline PatternFinding.py]: ` return not ` is used if the statement is true but output needs to be false, which is not the case here.
Only ` return ` would do here.

File: PatternFinding.py [hunk 2]
```diff
@@ -161,11 +156,7 @@ def IsPossibleQRContour(self, contourindex, nooflevels):
         if (level >= nooflevels):
             print level
             IsAreaSame = self.CheckingRatioOfContours(contourindex)
-            if IsAreaSame is True:
-                return True
-            else:
-                return False
-            return True
+            return not (IsAreaSame is True)
```
Comment (Griffintaur) [inline PatternFinding.py]: above comment is applicable here too.

File: PatternFinding.py (hunk 1 above)
Comment (debugduck) [reply PatternFinding.py]: Fixed.

File: PatternFinding.py (hunk 2 above)
Comment (debugduck) [reply PatternFinding.py]: Fixed, sorry about that!

**Comment by Griffintaur** [comment]
Thanks, @debugduck  for contributing.

**Pull request merged**